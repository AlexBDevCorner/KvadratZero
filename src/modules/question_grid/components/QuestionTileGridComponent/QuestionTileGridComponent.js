import React from 'react'

export default ({ questionList }) => (
  <div>

  </div>
)
