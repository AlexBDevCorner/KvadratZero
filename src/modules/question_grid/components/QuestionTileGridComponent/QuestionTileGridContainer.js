import React, { Component } from 'react'

import QuestionTileGridComponent from './QuestionTileGridComponent'

export class QuestionTileGridContainer extends Component {

  render() {
    return (
      <QuestionTileGridComponent />
    )
  }

}
