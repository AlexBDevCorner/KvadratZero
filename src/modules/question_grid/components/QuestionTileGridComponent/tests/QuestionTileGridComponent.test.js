import React from 'react'
import { shallow } from 'enzyme'

import QuestionTileGridComponent from '../QuestionTileGridComponent'

const component = shallow(<QuestionTileGridComponent />)

describe('QuestionTileGridComponent tests :', () => {

  it('should render properly', () => {
    expect(component).toMatchSnapshot()
  })

})
