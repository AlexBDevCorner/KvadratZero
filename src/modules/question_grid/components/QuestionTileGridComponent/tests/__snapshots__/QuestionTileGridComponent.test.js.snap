// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionTileGridComponent tests : should render properly 1`] = `<div />`;
